Dockerfile

```diff
@@ -49,6 +49,9 @@ RUN git clone https://github.com/indralab/mira.git /home/jupyter/mira && \
     pip install --no-cache-dir /home/jupyter/mira/"[ode,tests,dkg-client,sbml]" && \
     rm -r /home/jupyter/mira
 
+# Install Gilda - Mira Dependency 
+RUN pip install gilda
```

Comment:
Do we know if theres a version we should pin? I'm good with this as is, but realize at some point in the future it could break down...